### PR TITLE
feat!: Rename project input variable to project_id

### PR DESCRIPTION
**Pull request by q2w**
This PR makes below changes,

1. Renames project to project_id input variable to standardize across terraform blueprints
2. Adds output variable visibility in UI metadata
3. Update connection metadata version to match SemVer format.

### CODE REVIEW

File: metadata.yaml
Comment (ayushmjain) [inline metadata.yaml:1]: Please update project_id here as well.

Comment (q2w) [reply metadata.yaml:1]: Done.

**Review by ayushmjain — APPROVED** [review]

**Review by apeabody — APPROVED** [review]
Thanks @q2w @ayushmjain!

LGTM

**Comment by apeabody** [comment]
Thanks for including the upgrade guide!

**Pull request merged**